about.html

```diff
@@ -0,0 +1,151 @@
+            Normal in all Business related fields like - E-commerce - Marketing and Sales - Digital transformation -
+            Lean management and Business development - Investment - Opportunities for entrepreneurs - Supply Chain and
+            Logistics
+        </p>
+        <p>
+            Please contcat us per Email for any further information
+            about StartUp Land Summit 2022
+
+        </p>
+        <p class="header__Email">info@startuplandegypt.com</p>
+    </header>
+
+    <!-- main section -->
+    <main class="main" id="program">
+        <div class="main__heading">
+            <h2>Creative commons </h2>
+            <div class="main__line"></div>
+            <p>The Logo of startup land summit 2022 was decieded through logo design competition from 1.November to
+                5.December</p>
+        </div>
+        <div class="main__logo">
+            <img src="./assets/startup.jpg" alt="satrtup" />
+        </div>
+
+    </main>
+
+    <!-- times section -->
+    <section class="time">
+
+        <h2> See the past startup Global Summits</h2>
+        <div class="main__heading">
+            <div class="main__line"></div>
+        </div>
+        <p class="time__p">
+            Take a look at the last two startup
+            land Summits took place in the greek campus
+            and american university
+        </p>
+        <div class="time__container">
+            <div class="time__one">
+                <div class="time__text">
+                    <h2>2020</h2>
+                    <p>Startup land summit 2020 in American University</p>
+                </div>
+            </div>
+
+            <div class="time__two">
+                <div class="time__text">
+                    <h2>2019</h2>
+                    <p>Startup land summit 2019 in Greek campus</p>
+
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- DESKTOP: partner section -->
+    <section class="partner" id="sponsor">
+        <h2>Partner</h2>
+        <div class="main__heading">
+            <div class="main__line"></div>
+        </div>
+        <div class="partner__logos">
+            <span class="partner__logos__type1">IBM</span>
+            <span class="partner__logos__type2">Microsoft</span>
+            <span class="partner__logos__type2">Amideast</span>
+            <span class="partner__logos__type1">S PLus</span>
+            <span class="partner__logos__type1">USAID</span>
+            <span class="partner__logos__type2">Nokia</span>
+        </div>
+
+    </section>
+
+    <!-- footer -->
+    <footer>
+        <div class="footer__container">
+            <img src="./assets/startup.jpg" alt="startup">
+            <div class="footer__container__content">
+                <p>@ 2021 startup land</p>
+                <p>Some Rights Reserved </p>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] As mentioned before, to match the original design better, there are some changes needed. In the desktop version, the footer has a dark background, with `color: white`, whereas in the mobile version it looks just like yours! You should consider implementing the changes on the desktop version.
- [ ] You should also consider cropping your logo image a bit (I see that there is an excess of white on the top and on the bottom of the image), right now, that is causing the footer to have a really big height, that doesn't really match the original design.
- [ ] One last thing, the writings should be almost aligned to the logo, so you should consider changing that as well!

Original design:
![image](https://user-images.githubusercontent.com/66967274/147093060-045398b5-cf76-40e6-9f32-6b934cfc38df.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/147093034-739c707e-d895-4730-8794-2962734d1c31.png)